pkg/controllers/dashboard/chart/chart_test.go

```diff
@@ -176,7 +176,7 @@ func TestGetChartValue(t *testing.T) {
 			configCache := fake.NewMockCacheInterface[*v1.ConfigMap](ctrl)
 			test.setup(configCache)
 			getter := chart.RancherConfigGetter{configCache}
-			got, err := getter.GetChartValues(tt.chartName)
```

Comment:
Unless you see any reason not to, could we either have:

```go
for _, tt := range tests {
    tt := tt
```

or

```go
for _, test := range tests {
    test := test
```
?

That way we don't risk using the "wrong" one in the future.

Also, unrelated, but can't wait for https://tip.golang.org/wiki/LoopvarExperiment to be released so we no longer have to use this trick.

Reply:
This was author's preference. Although I personally think that either of suggested variants would be better it's not this PR's intent to change that preference. And it's not the only place where this naming was used.

Reply:
> Also, unrelated, but can't wait for https://tip.golang.org/wiki/LoopvarExperiment to be released so we no longer have to use this trick.
Agreed, and we can already use it with Go 1.21 (not by default though).